src/util/languageUtils.js

```diff
@@ -38,16 +38,10 @@ export const getLanguage = (
 	request,
 	supportedLangs,
 	defaultLang = LANG_DEFAULT
-) => {
-	// return the first language hit in the order of preference
-	supportedLangs.sort(l => l !== defaultLang);
```

Comment:
Ah I see. I was reading the sort API but not really following what this sort was intended to do. I agree that a comment update would be helpful here as well to explain the sort and `defaultLang`